src/vs/workbench/contrib/inlineChat/browser/media/inlineChat.css

```diff
@@ -72,6 +72,15 @@
 	outline: none;
 }
 
+.inline-chat-widget,
+.terminal-inline-chat {
```

Comment:
You can't reference `.terminal-inline-chat` from here as it's a soft layer breaker.  Try prefix the rule with something like `.monaco-workbench .inline-chat .chat-widget` which should work in both.

Reply:
👍🏻 that worked. for some reason didn't see `inline-chat` in regular inline chat flow at first